release.sh

```diff
@@ -47,7 +47,8 @@ sbt '+publishSigned'
 
 echo "Publishing docs..."
 sbt makeSite
-mv documentation/target/sphinx/html ../releasing-the-docs
+rm -rf documentation/target/paradox/paradox # Paradox site bug?
```

Comment:
Best to report on paradox repo I think. Issue is that it should have cleaned up that directory by itself right?